artifacts/examples/example-tapp.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps.tkestack.io/v1
+apiVersion: tke.cloud.tencent.com/v1
```

Comment:
Why do we need change the group name?

Reply:
I'd deployed tapp controller via [tapp-controller.yaml](https://github.com/tkestack/tapp/blob/master/deployment/tapp-controller.yaml)

![image](https://user-images.githubusercontent.com/6958477/100420491-87cb4880-30c1-11eb-8a7b-6f5dc1a6c770.png)

Reply:
Sorry, the CRD in tapp-controller.yaml is wrong. It will be better change it.